index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+
+  <div>
+    <header id="header">
+      <h2> Handcrafted, home-made masterpieces </h2>
+      <div>
+        <img id="header-img"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+          alt="original trombones logo">
+      </div>
+
+      <div>
+        <nav id="nav-bar">
+          <a href="#How-It-Works" class="nav-link">
+            How It Works</a>
```

Comment:
- [ ] Well done on the workflow so far, but you should practice good indentation for this will make the code look clean.

Reply:
- [ ] You did a great job adding comments to code and you are being really polite but kindly be more specific about what are you requesting. When the student will read your review he/she has to understand everything you are saying.